recipes/qwt/6.2.0/conanfile.py

```diff
@@ -0,0 +1,100 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps, cmake_layout
+from conan.tools.files import apply_conandata_patches, copy, get
+from conan.tools.build import cross_building
+from conan.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.50"
+
+class QwtConan(ConanFile):
+    name = "qwt"
+    license = "LGPL-2.1-or-later"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://qwt.sourceforge.io/"
+    topics = ("conan", "archive", "compression")
+    description = (
+        "The Qwt library contains GUI Components and utility classes which are primarily useful for programs "
+        "with a technical background. Beside a framework for 2D plots it provides scales, sliders, dials, compasses, "
+        "thermometers, wheels and knobs to control or display values, arrays, or ranges of type double."
+    )
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "plot": [True, False],
+        "widgets": [True, False],
+        "svg": [True, False],
+        "opengl": [True, False],
+        "designer": [True, False],
+        "polar": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "plot": False,
+        "widgets": False,
+        "svg": False,
+        "opengl": False,
+        "designer": False,
```

Comment:
```suggestion
        "plot": True,
        "widgets": True,
        "svg": False,
        "opengl": True,
        "designer": True,
```
Let's be consistent with https://github.com/conan-io/conan-center-index/blob/master/recipes/qwt/all/conanfile.py#L31

Reply:
with designer at True, I get following error:

```
CMake Error at designer/CMakeLists.txt:14 (add_library):
  Target "qwt_designer_plugin" links to target "Qt5::Designer" but the target
  was not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?
```

I don't find which option to enable to have this target.

Reply:
If I try to add option as ""qt:qttools": True" I get following error on qt build:

```
qt/5.15.5: Configuring sources in /home/codespace/.conan/data/qt/5.15.5/_/_/source
qt/5.15.5: 
ERROR: qt/5.15.5: Error in source() method, line 423
	get(self, **self.conan_data["sources"][self.version],
	ReadError: file could not be opened successfully:
- method gz: ReadError('not a gzip file')
- method bz2: ReadError('not a bzip2 file')
- method xz: CompressionError('lzma module is not available')
- method tar: ReadError('invalid header')
```